templates/about/index.tpl

```diff
@@ -12,16 +12,16 @@
 {assign var="pageTitle" value="about.aboutTheJournal"}
 {include file="common/header.tpl"}
 {/strip}
-<div id="aboutPeople">
+<div id="people">
 <h3>{translate key="about.people"}</h3>
 <ul>
 	{if not (empty($journalSettings.mailingAddress) && empty($journalSettings.contactName) && empty($journalSettings.contactAffiliation) && empty($journalSettings.contactMailingAddress) && empty($journalSettings.contactPhone) && empty($journalSettings.contactFax) && empty($journalSettings.contactEmail) && empty($journalSettings.supportName) && empty($journalSettings.supportPhone) && empty($journalSettings.supportEmail))}
-		<li><a href="{url op="contact"}">{translate key="about.contact"}</a></li>
+		<li id="contact"><a href="{url op="contact"}">{translate key="about.contact"}</a></li>
 	{/if}
-	<li id="editorialTeamLink"><a href="{url op="editorialTeam"}">{translate key="about.editorialTeam"}</a></li>
+	<li id="editorialTeam"><a href="{url op="editorialTeam"}">{translate key="about.editorialTeam"}</a></li>
 	{if $peopleGroups}
 		{iterate from=peopleGroups item=peopleGroup}
-			<li><a href="{url op="displayMembership" path=$peopleGroup->getId()}">{$peopleGroup->getLocalizedTitle()|escape}</a></li>
+			<li id="{$peopleGroup->getId()}"><a href="{url op="displayMembership" path=$peopleGroup->getId()}">{$peopleGroup->getLocalizedTitle()|escape}</a></li>
```

Comment:
I would suggest prefixing this with something, rather than just using the numeric ID.